front-end/src/app/reports/transactions/transaction-list/transaction-list-table-base.component.ts

```diff
@@ -45,7 +45,8 @@ export abstract class TransactionListTableBaseComponent extends TableListBaseCom
         transaction.itemized === false &&
         this.reportIsEditable &&
         !transaction.parent_transaction &&
-        !transaction.parent_transaction_id,
+        !transaction.parent_transaction_id &&
+        !['C', 'D'].includes(transaction.transactionType.scheduleId),
```

Comment:
a shame we don't have constants for these

Reply:
an enum, ScheduleIds, has been added